tests/Feature/Engine/IntermediateMessageEventTest.php

```diff
@@ -102,28 +102,28 @@ public function createMessageIntermediateEventProcesses()
      */
     public function createSignalIntermediateEventProcesses()
     {
-        $signal = $this->factory->createInstanceOf(SignalInterface::class);
+        $signal = $this->repository->createSignal();
         $signal->setId('Signal1');
         $signal->setName('SignalName');
 
         //Process A
-        $processA = $this->factory->createInstanceOf(ProcessInterface::class);
-        $startA = $this->factory->createInstanceOf(StartEventInterface::class);
-        $activityA = $this->factory->createInstanceOf(ActivityInterface::class);
-        $eventA = $this->factory->createInstanceOf(IntermediateThrowEventInterface::class);
-        $signalEventDefA = $this->factory->createInstanceOf(SignalEventDefinitionInterface::class);
+        $processA = $this->repository->createProcess();
+        $startA = $this->repository->createStartEvent();
+        $activityA = $this->repository->createActivity();
+        $eventA = $this->repository->createIntermediateThrowEvent();
+        $signalEventDefA = $this->repository->createSignalEventDefinition();
```

Comment:
Replace the usage of factory::createInstanceOf by repository::createBPMNElement()